### ISSUE TITLE
Soft crash when opening achievements 

### ISSUE THREAD

**Issue by Hauki** [playgameservices/play-games-plugin-for-unity#2545]
When opening achievements the app reports an error (ShowAchievementsUI). It still shows the achievements to the users.  

`NullReferenceException A null value was found where an object instance was required. 
    unknown file GooglePlayGames.PlayGamesPlatform+<ReportProgress>c__AnonStorey3.<>m__0
    unknown file GooglePlayGames.OurUtils.PlayGamesHelperObject.Update`

Using Unity 2017.4.25f1
GPGS 0.9.63

Only happens if some of the achievements have progress.

**Comment by toxikman** [comment]
We're getting the same issue, also with Unity 2017.4.25f1, GPGS 0.9.63

```
NullReferenceException: A null value was found where an object instance was required.
GooglePlayGames.PlayGamesPlatform+<ReportProgress>c__AnonStorey3.<>m__0 (GooglePlayGames.BasicApi.Achievement[] ach)
GooglePlayGames.OurUtils.PlayGamesHelperObject.Update ()
```
Got it on an Android 8 device.

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.9.64
